scheduler/utilization_based_host_allocator.go

```diff
@@ -120,19 +120,6 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData *HostA
 	}
 	numNewHostsToRequest := numNewHostsRequired + numAdditionalHostsToMeetMinimum
 
-	grip.Info(message.Fields{
```

Comment:
I took a look and it looks like this + the `"queue state report"` one are leftovers from SRE dashboards, which seem unused since they don't render properly.